ANTLR/src/Compiler/CodeGeneration/JavaScriptCodeGenerationVisitor.java

```diff
@@ -210,7 +215,10 @@ public String visitExprFunccall(ExprFunccallContext ctx) {
      */
     @Override
     public String visitExprFunccallPrint(ExprFunccallPrintContext ctx) {
-        String exprParams = visit(ctx.funcCall().exprParams());
+        String exprParams = "";
+        if(ctx.funcCall().exprParams() != null){
+            exprParams = visit(ctx.funcCall().exprParams());
+        }
```

Comment:
Det her er vidst et gammelt commit, som har sneget sig med. Men her er den samme rettelse som sidst:

```suggestion
        if (ctx.funcCall().exprParams() != null)
            exprParams = visit(ctx.funcCall().exprParams());
```